spec.bs

```diff
@@ -5398,7 +5403,7 @@ To <dfn>look up per-buyer multi-bid limit</dfn> given an [=auction config=] |auc
 <h3 id=bid-generators>Bid generator</h3>
 
 A <dfn>per buyer bid generator</dfn> is an [=ordered map=] whose [=map/keys=] are [=URLs=]
-representing [=interest group/trusted bidding signals urls=], and whose [=map/values=] are
+or null representing [=interest group/trusted bidding signals urls=], and whose [=map/values=] are

```

Comment:
I can't unresolve the old thread: https://github.com/WICG/turtledove/pull/1180#discussion_r1605439214

Could we add a Note: to the spec to clarify why the map can have null keys?

Reply:
Oh, sorry, misunderstood the relation between your two suggestions, one sec.

Reply:
And added a sentence

Reply:
SGTM